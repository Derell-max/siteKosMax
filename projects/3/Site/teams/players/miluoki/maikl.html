<html>
 <head>
 <meta charset="Windows-1251">
 <title>Майкл Картер-Уильямс</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/miluoki.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/miluoki/4.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">67</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">446</font></td>
     <td><font face="impact" color="white">6.7</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">64</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">292</font></td>   
     <td><font face="impact" color="white">4.4</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">2179:20</font></td>
     <td><font face="impact" color="white">32:32</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">66</font></td>
     <td><font face="impact" color="white">1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">980</font></td>
     <td><font face="impact" color="white">14.6</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">358</font></td> 
     <td><font face="impact" color="white">5.3</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">329/769</font></td>
     <td><font face="impact" color="white">4.9/11.5</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">111</font></td> 
     <td><font face="impact" color="white">1.7</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">42.8%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">256</font></td>  
     <td><font face="impact" color="white">3.8</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">40/162</font></td>
     <td><font face="impact" color="white">0.6/2.4</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">31</font></td>
     <td><font face="impact" color="white">0.5</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">24.7%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">73</font></td>
     <td><font face="impact" color="white">1.1</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">202/291</font></td>
     <td><font face="impact" color="white">3/4.3</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">172</font></td>   
     <td><font face="impact" color="white">2.6</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Майкл Картер-Уильямс</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">69.4%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">774</font></td>  
     <td><font face="impact" color="white">11.6</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>