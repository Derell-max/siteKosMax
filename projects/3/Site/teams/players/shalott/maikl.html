<html>
 <head>
 <meta charset="Windows-1251">
 <title>Майкл Кидд-Гилкрист</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/shalott.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/shalott/13.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">55</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">77</font></td>
     <td><font face="impact" color="white">1.4</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">52</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">307</font></td>   
     <td><font face="impact" color="white">5.6</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">1587:26</font></td>
     <td><font face="impact" color="white">28:52</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">109</font></td>
     <td><font face="impact" color="white">2</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">598</font></td>
     <td><font face="impact" color="white">10.9</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">416</font></td> 
     <td><font face="impact" color="white">7.6</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">224/482</font></td>
     <td><font face="impact" color="white">4.1/8.8</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">30</font></td> 
     <td><font face="impact" color="white">0.5</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">46.5%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">63</font></td>  
     <td><font face="impact" color="white">1.1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">0/0</font></td>
     <td><font face="impact" color="white">0/0</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">38</font></td>
     <td><font face="impact" color="white">0.7</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">0%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">63</font></td>
     <td><font face="impact" color="white">1.1</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">150/214</font></td>
     <td><font face="impact" color="white">2.7/3.9</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">115</font></td>   
     <td><font face="impact" color="white">2.1</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Майкл Кидд-Гилкрист</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">70.1%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">596</font></td>  
     <td><font face="impact" color="white">10.8</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>