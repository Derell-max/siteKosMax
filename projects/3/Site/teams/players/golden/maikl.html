<html>
 <head>
 <meta charset="Windows-1251">
 <title>Джеймс Майкл МакАду</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/golden.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/golden/14.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">15</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">2</font></td>
     <td><font face="impact" color="white">0.1</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">22</font></td>   
     <td><font face="impact" color="white">1.5</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">137:09</font></td>
     <td><font face="impact" color="white">9:09</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">15</font></td>
     <td><font face="impact" color="white">1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">62</font></td>
     <td><font face="impact" color="white">4.1</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">37</font></td> 
     <td><font face="impact" color="white">2.5</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">24/44</font></td>
     <td><font face="impact" color="white">1.6/2.9</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">5</font></td> 
     <td><font face="impact" color="white">0.3</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">54.5%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">6</font></td>  
     <td><font face="impact" color="white">0.4</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">0/0</font></td>
     <td><font face="impact" color="white">0/0</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">9</font></td>
     <td><font face="impact" color="white">0.6</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">0%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">3</font></td>
     <td><font face="impact" color="white">0.2</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">14/25</font></td>
     <td><font face="impact" color="white">0.9/1.7</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">21</font></td>   
     <td><font face="impact" color="white">1.4</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Джеймс Майкл МакАду</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">56%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">54</font></td>  
     <td><font face="impact" color="white">3.6</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>